src/context/GalleryModalContext.tsx

```
@@ -0,0 +1,79 @@
+  const openModal = useCallback((clickedItem: GalleryModalItem) => {
+    const allItems = buildItemsFromDom()
+    const index = allItems.findIndex(i => i.src === clickedItem.src)
+    setItems(allItems)
+    setCurrentIndex(index >= 0 ? index : 0)
+  }, [])
+
+  // expose to global for vanilla trigger
+  useEffect(() => {
+    ;(window as any).openGalleryModal = (item: GalleryModalItem) => openModal(item)
+    return () => {
+      delete (window as any).openGalleryModal
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)